packages/zone.js/test/npm_package/npm_package.spec.ts

```diff
@@ -86,6 +86,12 @@ describe('Zone.js npm_package', () => {
           expect(shx.cat('zone.umd.js')).not.toContain('sourceMappingURL');
         });
       });
+
+      it('zone.js(es5) should contain use strict', () => {
+        checkInSubFolder('./bundles', () => {
+          expect(shx.cat('zone.umd.js')).toContain('use strict');
```

Comment:
Consider making this test (and the es2015 one) more strict by checking for `'use strict';` and possibly that the file starts with that. Just `toContain('use strict')` could easily catch other occurrences of the words, which are not related to the `'use strict'` pragma.

For example:

```suggestion
          expect(shx.cat('zone.umd.js')).toMatch(/^\s*'use strict';/);
```

Reply:
Thanks, I have updated this one.